tests/destroyed_symlinks_test.py

```diff
@@ -0,0 +1,80 @@
+import os
+from subprocess import check_call
+from subprocess import check_output
+
+import pytest
+
+from pre_commit_hooks.destroyed_symlinks import find_destroyed_symlinks
+from pre_commit_hooks.destroyed_symlinks import main
+from pre_commit_hooks.destroyed_symlinks import normalize_content
+
+TEST_SYMLINK = 'test_symlink'
+
+
+@pytest.fixture
+def repo_with_destroyed_symlink(tmpdir):
+    source_repo = tmpdir.join('src')
+    os.makedirs(source_repo, exist_ok=True)
+    test_repo = tmpdir.join('test')
+    with source_repo.as_cwd():
```

Comment:
note that you can use `git -C source_repo ...` instead of `as_cwd()`

Reply:
I know but this way it's a bit more readable IMO.